### ISSUE TITLE
Ignore character entity representation

### ISSUE THREAD

**Issue by con-cis** [intlify/eslint-plugin-vue-i18n#481]
**Please describe what the rule should do:**

**What category should the rule belong to?**
Ignore Character entity representations like '•', unicode equivalent 'U+2022'

[ ] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<span v-else class="width-6 text-center">&bull;</span>
```

```vue
<span v-if="!isCalendarRef">&ndash; {{ commonLabels.noItems }} &ndash;</span>
```
